public/app/views/pdf/_digital_object_links.html.erb

```diff
@@ -0,0 +1,12 @@
+<% Array(instances).each do |instance| %>
+    <% next if instance['_inherited'] %>
+    <% if instance['instance_type'] == 'digital_object' && instance['digital_object'] %>
+        <% file_versions = instance['digital_object']['_resolved']['file_versions'] %>
+        <% Array(file_versions).each do |file| %>
+            <% if file['file_uri'] && file['use_statement'] != 'image-thumbnail' %>
```

Comment:
Thanks @fordmadox, that's an easy enough change.  I've just pushed that up now.

@lmcglohon This one's probably ready for merge if you're happy.  Thanks!